Sources/Sentry/SentryCrashWrapper.m

```diff
@@ -13,14 +13,6 @@
 
 @implementation SentryCrashWrapper
 
-+ (instancetype)sharedInstance
```

Comment:
`m`: I think this change could be in an extra PR as it makes sense without all the other changes. Even if we agree not to proceed with this PR, this change would make it to the main branch.

Reply:
see https://github.com/getsentry/sentry-cocoa/pull/2941